lms/templates/admin/base.html.jinja2

```diff
@@ -0,0 +1,85 @@
+<!DOCTYPE html>
+<html>
+  <head>
+    <meta charset="utf-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+    <title>{% block title %}LMS Admin{% endblock %}</title>
+    <link rel="stylesheet" href="https://cdn.jsdelivr.net/npm/bulma@0.9.2/css/bulma.min.css">
+  </head>
+  <body>
+
+  {% block base_navbar %}
+  <nav class="navbar" role="navigation" aria-label="main navigation">
+  <div class="navbar-brand">
+    <a class="navbar-item" href="{{ request.route_url("admin.index") }}">
+      <p class="is-size-4">ADMIN</p>
+    </a>
+  </div>
+
+  <div id="navbarBasicExample" class="navbar-menu">
+    <div class="navbar-start" style="flex-grow: 1; justify-content: center;">
+      <a class="navbar-item" 
+          href="{{ request.route_url("admin.installations") }}" >
+        Installations
+      </a>
+
+    <div class="navbar-end">
+      <div class="navbar-item">
+        <div class="buttons">
+          <a class="button is-light" href="{{ request.route_url("pyramid_googleauth.logout") }}" >
+            Log out
+          </a>
```

Comment:
The log out button seems to disappear if the window is too thin :)

Reply:
Responsive design :stuck_out_tongue: